roles-in-tech/developer-advocate.md

```diff
@@ -2,89 +2,146 @@
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+- Lesson: 10 min
+- Guided Practice: 30 min
+- Independent Practice: 10 min
 
-### Prerequisites
 
-Here are links to lessons that should be completed before this lesson:
-
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+A developer evangelist is a spokesperson, mediator and translator between a company and both its technical staff and outside developers. Inorder to be a developer advocate one must have the following skills:
+First there are the necessary skills:
+- Great development skills — both in creating and explaining software and products
+- Excellent communication skills — this job is about reaching out, listening and distilling information
+- Excellent networking skills — you’re meant to collect contacts and keep them happy
+Why take on this job?
+Considering how full your plate is, the following question should come up:
+Why would you want to become a developer evangelist (or developer advocate)?
+There are perks to being someone in this role. Mostly that it is a natural progression for a developer who wants to move from a delivery role to one where they can influence more what the company does.
+- It bridges the communication gap — developers have a bad reputation when it comes to communication. Showing that someone technical can help understanding each other is a good move for our market.
+- It helps avoiding frustration — a lot of engineering is not needed, but based on false assumptions or misguided “I want to use this”. Good evangelism helps building what is needed, not what is cool.
+- It bridges age and culture gaps — if you’re not interested in a cutthroat competition in engineering, you have a chance to use your talent otherwise.
+Before you jump on the opportunity, there are some very important points to remember:
+- Developer relations is not a starting position — most developer evangelists graduated from being developers in the same company. You need to know the pain to help prevent it.
+- There are part time opportunities though — engineers or people learning in the company can help with Devrel to ease into the job earlier.
+- Always be ready to prove your worth — measuring the impact of a developer evangelist is tough, you need to make sure you’re well organised in recording your successes.
+It’s a versatile, morphing and evolving role. 
 
-### Objectives
 
-**Participants will be able to:**
+### Objectives
 
-- leave this lesson knowing this
-- and how to do this on their own
-- Even more things
-- Even more things
+Developer Advocate is a software-focused technical person who has good communication and community-building skills.  The responsibilities of the role typically are some or all of the following:
```

Comment:
Again, awesome info, but “objectives” should just be a bulleted list of goals for the topic outline to achieve. Please move the rest under “lesson”.